.eslintrc

```diff
@@ -2,7 +2,8 @@
   "extends": "./packages/eslint-config-fb-strict/index.js",
   "parser": "babel-eslint",
   "rules": {
-    "no-multiple-empty-lines": 1
+    "no-multiple-empty-lines": 1,
+    "require-valid-file-annotation": 1
```

Comment:
I noticed that "flowtype/require-valid-file-annotation" didn't generate any warnings, so I'm concerned about using "flowtype/" prefix in this case.